roles/goaccess/templates/goaccess.service.j2

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Goaccess - fast web log analyzer and interactive viewer
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+Type=simple
+User={{ goaccess_user }}
+Group={{ goaccess_user }}
+Sockets=goaccess-input.socket
+StandardInput=socket
+ExecStart=/usr/bin/goaccess - \
+    --output {{ goaccess_report_root }}/index.html \
+    --real-time-html \
+    --persist
```

Comment:
**n.b.** I didn't put these options to the config file because `goaccess` can be used as a terminal viewer too. With these server options in the config, it would be impossible to use `goaccess` as a terminal viewer. Therefore, I decided to keep them here.